### DISCUSSION TITLE
iCub not moving after calibration

### DISCUSSION THREAD

**Discussion by cedricgoubard** [robotology/community#542] [Q&A] [answered]
Hello, 
I just calibrated our `iCubLondon01` following the instructions provided [here](https://github.com/robotology/icub-tech-support/files/7030963/TP_ROBOT_CALIBRATION.pdf). In short:
- I pressed the fault button
- I launched `yarprobotinterface`
- I launched `yarpmotorgui`
- I filled the `xls` file with the values for the different encoders
- I generated the `xml` files and placed them in `/exports/code/robotology-superbuild/build-icubunicorn/install/share/ICUBcontrib/robots/iCubLondon01/`, which is the repository returned by `yarp resource --from general.xml`
- I shut down the robot (`pc104` and motors), released the fault buttons, and turned it back on

Now, here are the issues I am facing:
- `iCub` does not move in standard position when I run `yarprobotinterface`
- I cannot move the motors using `yarpmotorgui`

Apart form that, everything seems to run correctly: `yarprobotinterface`, `iKinCartesianSolver` and `wholeBodyDynamics` all run, and the usual ports are created.

`yarprobotinterface` does raise warnings and errors, but I cannot really make sens of them. Here are [the logs](https://github.com/robotology/community/files/7056168/yarprobotinterface.log).

Here is what `yarpmotorgui` looks like:
![image](https://user-images.githubusercontent.com/40461909/130947202-993d62a2-2c0e-40ee-b4db-8c485e2b566c.png)
I tried changing the `idle` mode to `position`, but nothing happens.

Do you have any idea what might have gone wrong?
Any help would be greatly appreciated.

**Comment by pattacini** [comment, marked as answer]
Sorry @cedricgoubard 
But could you report this into robotology/icub-tech-support instead?
Thanks in advance.

**Reply by cedricgoubard** [reply]
No problem, I reported it https://github.com/robotology/icub-tech-support/issues/1182.
Sorry about that, since it is not a hardware issue, I thought it belonged here (c.f. [the new issue page on the icub-tech-support page](https://github.com/robotology/icub-tech-support/issues/new/choose)  below).
![image](https://user-images.githubusercontent.com/40461909/130963383-f35287eb-96d3-4173-9707-dafc823b70b2.png)

**Reply by pattacini** [reply]
Thanks heaps and never mind @cedricgoubard 👍🏻 

The HW maintenance has to be intended in a broader sense as per https://github.com/robotology/icub-tech-support/blob/master/.github/SUPPORT.md.
